BattleAIMaven/src/main/java/Visual/Animator.java

```diff
@@ -69,10 +72,18 @@ private void runMultiplayerHost(){
     private void runMultiplayerClient(){
         while(running){
             
-            EntityUpdateRequest newEntities;
+            //EntityUpdateRequest newEntities;
             try {
-                newEntities = (EntityUpdateRequest)ConnectionHandler.getInstance().readFromMatch();
-                panel.entityList = newEntities.gameEntities;
+                //newEntities = (EntityUpdateRequest)ConnectionHandler.getInstance().readFromMatch();
+                //panel.entityList = newEntities.gameEntities;  
+                
+                if(currentPacket != null && currentPacket.framesLeft() > 0)
+                    panel.entityList = currentPacket.consume();
```

Comment:
kek, i've just noticed i do panel.entityList = currentPacket.consume() in "if" and in the "else". Welp, this doesn't really deserve a commit.